### ISSUE TITLE
Page 163: Vim doesn't open after git merge

### ISSUE THREAD

**Issue by daldr-ntml** [gitlearningjourney/learning-git#1]
Hi, I hope this is an appropriate place to discuss this.

At p.163, step 3 says that the command line will enter Vim after the git merge command is executed. That is not my experience:

`$ git merge origin/main
Merge made by the 'ort' strategy.
 othercolors.txt | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 othercolors.txt
$`

I am using a git bash Terminal in VS Code.

**Comment by clairew98** [comment]
Hi David,

I have just done this exercise and found the same, also in VS Code. Does this cause any problems later in the book?

I'm wondering how we would be able to change the default commit message if we don't get the Vim editor as expected.

Did you find a workaround?

**Comment by amirmohammaddivani** [comment]
Hello guys, I am not a Git expert but I believe the "ort" strategy is basically the same as the fast-forward strategy but it's a bit more efficient. I guess Git decides which one to use and it usually uses "ort" because it's newer and more shiny. 
So the thing is that these two strategies do not require you to enter a commit message. If there are no conflicts during the merge process, and Git is able to complete the merge automatically using either the fast-forward or "ort" strategy, then you don't need to provide a commit message.
So basically what happens is that the branch you are merging into (the main branch) basically will point to another commit that the "feature" branch is/was pointing to, which is an existing commit with an ID, a message, etc. So no newer commits had to be made, hence no prompt for you to enter a commit message.
This is basically what I understand from Git and I could be wrong so research this on your own as well but I believe I'm right.
